app/views/liquids/_user_subscription.html.erb

```diff
@@ -0,0 +1,70 @@
+<div class="ltag__user-subscription-tag" id="user-subscription-tag" data-author-username=<%= author_username %>>
```

Comment:
Same as above, you can skip right over this file if you'd like :trollface:. It's all hacky to get the MVP UX running for design to polish up 🎉 .